src/javascript/app_2/Modules/Reports/Containers/statement.jsx

```diff
@@ -0,0 +1,107 @@
+import { PropTypes as MobxPropTypes }       from 'mobx-react';
+import PropTypes                            from 'prop-types';
+import React                                from 'react';
+import { withRouter }                       from 'react-router-dom';
+import { localize }                         from '_common/localize';
+import DataTable                            from 'App/Components/Elements/DataTable';
+import { getContractPath }                  from 'App/Components/Routes/helpers';
+import { connect }                          from 'Stores/connect';
+import { getStatementTableColumnsTemplate } from '../Constants/data-table-constants';
+import PlaceholderComponent                 from '../Components/placeholder-component.jsx';
+import { ReportsMeta }                      from '../Components/reports-meta.jsx';
+import EmptyTradeHistoryMessage             from '../Components/empty-trade-history-message.jsx';
+
+class Statement extends React.Component {
+    componentDidMount()    { this.props.onMount(); }
+    componentWillUnmount() { this.props.onUnmount(); }
```

Comment:
This seems inconsistent throughout the files, do we want to use one liners or do we want to expand them?